draft-ietf-quic-recovery.md

```diff
@@ -752,6 +752,12 @@ twice the maximum datagram size. This follows the analysis and recommendations
 in {{?RFC6928}}, increasing the byte limit to account for the smaller 8 byte
 overhead of UDP compared to the 20 byte overhead for TCP.
 
+Prior to validating the client's address, the server can also be limited by
```

Comment:
How about saying "can be limited further"? I think that might better clarify that it is a `min` of the two.

Reply:
SGTM